src/Ng/Jobs/BackupV3Storage.cs

```diff
@@ -0,0 +1,236 @@
+                   + $"[-{Arguments.DestStorageAccountName} <azure-acc> "
+                   + $"-{Arguments.DestStorageKeyValue} <azure-key> "
+                   + $"-{Arguments.DestStorageContainer} <azure-container>] "
+                   + $"-{Arguments.BackupDays} <backup Days>] "
+                   + $"[-{Arguments.VaultName} <keyvault-name> "
+                   + $"[-{Arguments.VaultName} <keyvault-name> "
+                   + $"-{Arguments.ClientId} <keyvault-client-id> "
+                   + $"-{Arguments.CertificateThumbprint} <keyvault-certificate-thumbprint> "
+                   + $"[-{Arguments.ValidateCertificate} true|false]]";
+        }
+
+        protected override void Init(IDictionary<string, string> arguments, CancellationToken cancellationToken)
+        {
+            _luceneStorageAccountName = arguments.GetOrThrow<string>(Arguments.LuceneStorageAccountName);
+            _luceneStorageKeyValue = arguments.GetOrThrow<string>(Arguments.LuceneStorageKeyValue);
+            _luceneStorageContainer = arguments.GetOrThrow<string>(Arguments.LuceneStorageContainer);
+
+            _catalogStorageAccountName = arguments.GetOrThrow<string>(Arguments.CatalogStorageAccountName);
+            _catalogStorageKeyValue = arguments.GetOrThrow<string>(Arguments.CatalogStorageKeyValue);
+            _catalogStorageContainer = arguments.GetOrThrow<string>(Arguments.CatalogStorageContainer);
+
+            _registrationStorageAccountName = arguments.GetOrThrow<string>(Arguments.RegistrationStorageAccountName);
+            _registrationStorageKeyValue = arguments.GetOrThrow<string>(Arguments.RegistrationStorageKeyValue);
+            _registrationStorageContainer = arguments.GetOrThrow<string>(Arguments.RegistrationStorageContainer);
+            _registrationgzStorageContainer = arguments.GetOrThrow<string>(Arguments.RegistrationGZStorageContainer);
+
+            _destStorageAccountName = arguments.GetOrThrow<string>(Arguments.DestStorageAccountName);
+            _destStorageKeyValue = arguments.GetOrThrow<string>(Arguments.DestStorageKeyValue);
+
+            _backupDays = arguments.GetOrDefault<int>(Arguments.BackupDays, DefaultBackupDays);
+        }
+
+        protected override async Task RunInternal(CancellationToken cancellationToken)
+        {
+            List<Task> deleteTasks = new List<Task>();
+            DateTime currentDate = DateTime.UtcNow;
+
+            deleteTasks.Add(TryDeleteContainerAsync(_luceneStorageContainer, currentDate, _destStorageAccountName, _destStorageKeyValue));
+            deleteTasks.Add(TryDeleteContainerAsync(_catalogStorageContainer, currentDate, _destStorageAccountName, _destStorageKeyValue));
+            deleteTasks.Add(TryDeleteContainerAsync(_registrationStorageContainer, currentDate, _destStorageAccountName, _destStorageKeyValue));
+            deleteTasks.Add(TryDeleteContainerAsync(_registrationgzStorageContainer, currentDate, _destStorageAccountName, _destStorageKeyValue));
```

Comment:
There's a little inconsistency w/ some APIs taking destination storage info, and others assuming it. For instance, TryDelete and TryCreate have destination storage args, whereas TryCopy doesn't. Maybe just name all to clarify that they work against dest storage, and only take in the dest container name?